docs/en/observability/get-started.asciidoc

```diff
@@ -0,0 +1,32 @@
+[chapter, role="xpack"]
+[[install-observability]]
+= Get started
+
+To use the Elastic Observability, you need {ess} for storing and searching your 
+data, and {kib} for visualizing and managing it.
+
+[float]
+[[set-up-on-cloud]]
+== Set up on Cloud
+
+To get started quickly, you can spin up a deployment of our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]. The deployment includes
```

Comment:
```suggestion
{ess-trial}[hosted {ess}]. The deployment includes
```

Reply:
This sentence was removed as per one of your previous comments.